setup.py

```diff
@@ -16,31 +16,44 @@
 # We use the version to construct the DOWNLOAD_URL.
 DOWNLOAD_URL = ''.join((REPO_URL, '/tarball/release/', VERSION))
 
-REQUIREMENTS = ['pystache', 'Sphinx','kruxstatsd', 'argparse', 'lockfile', 'future']
+REQUIREMENTS = ['pystache', 'kruxstatsd', 'lockfile', 'future']
 if os.name == 'posix' and sys.version_info[0] < 3:
     # For Python 2.*, install the backported subprocess
     REQUIREMENTS.append('subprocess32')
 
+TEST_REQUIREMENTS = ['coverage', 'mock', 'nose']
+
+LINT_REQUIREMENTS = ['flake8']
+
 setup(
     name='krux-stdlib',
     version=VERSION,
-    author='Paul Lathrop',
-    author_email='paul@krux.com',
-    maintainer='Paul Lathrop',
-    maintainer_email='paul@krux.com',
+    author='Salesforce DMP Platform Engineering',
+    author_email='krux-pe@salesforce.com',
     description='Standard library of python modules used at Krux.',
     long_description="""
     Standard library of python modules used at Krux. These modules are written
     to make it easy to do things the Krux Way in your Python applications.
     """,
     url=REPO_URL,
     download_url=DOWNLOAD_URL,
-    license='License :: OSI Approved :: MIT License',
-    packages=find_packages(),
-    install_requires=REQUIREMENTS,
-    tests_require=[
-        'coverage',
-        'mock',
-        'nose',
+    license='MIT',
```

Comment:
Had we made a decision to continue supporting krux-stdlib as a FOSS project? It seems like a grab-bag of DMP-specific classes & functions.

Reply:
As far as I know, it was an MIT licensed project. While whether the other people want this or not may vary, but there is no reason to turn this into a closed source project.